dataland-frontend/src/components/forms/parts/fields/NaceCodeFormField.vue

```diff
@@ -40,9 +40,27 @@ export default defineComponent({
   },
   data() {
     return {
-      selectedNaceCodes: [] as Array<string>,
```

Comment:
Why do we need this?

Reply:
The value is already stored in the parent if necessary, right?

Reply:
Resolving this, because it turns out that this variable is needed to store the currently chosen nace codes.